### DISCUSSION TITLE
Navigating back to homepage breaks plugins

### DISCUSSION THREAD

**Discussion by c9obvi** [squidfunk/mkdocs-material#6246] [Q&A]
Hello! New insider here, thank you for all the great tooling. My question/bug:
When I initially hid the home page menu/column on the left, that hide feature would become undone when navigating to another page and navigating back to home (without refreshing the page).

I resolved this with some JS and got a full width hero section as intended and it remains that way when navigating away from and back to the home page. (index.md)

Now, as an insiders we wanted to use the grid card feature. Inserting the grid code as presented in documentation just resulted in plain text so I resorted to coding out html <ul> and coding out the list that made the cards + CSS icons etc (pretty manual but no biggie -- I am sure I messed up something that wrote over your syntax provided in documentation. 

Anyway, I got that working but now the cards are having that issue where once you navigate away from index.md and back via the logo or home button on the nav bar... the cards do not work and the icons in the card become external links that take you to the image when you click on them.

Here is the current dev link. You can see intended function with initial load, but after clicking anywhere on any of the cards and being redirected they no longer function when you navigate back to home. I must be doing something silly. Thank you all for hour help!

https://development--3dr-docs.netlify.app/

and here is the actual code: https://gitlab.com/3dr-eng/knowledgebase.git

**Comment by squidfunk** [comment]
So it seems that you have a lot of customizations in place, and the problems you are talking about are in those customizations. I, personally, can't assist you with making them work, because I need to focus on the project itself, benefiting all users, but maybe another user from the community can help? ☺️

**Comment by c9obvi** [comment]
No worries, thought maybe it would be something you had heard of. Hopefully someone from the community chimes in, thank you!
